src/model.rs

```diff
@@ -74,7 +78,9 @@ impl CxxLogEvent {
     }
 
     pub fn fields(&self) -> Vec<String> {
-        self.log_event.keys().map(|s| s)
-            .expect("log event should have some fields").collect()
+        self.log_event.keys()
```

Comment:
here we need to replace "\." with ".", because vector replace "." with "\." when making path.